### ISSUE TITLE
Authorization Code Grant still not redirecting to custom protocols.

### ISSUE THREAD

**Issue by AndyIbanez** [joshstar/AniList-API-Docs#110]
I'm sure I have reported this before, but not on its own issue... I can't seem to find it now.

I'm doing tests and I have set the Client Redirect Uri to be `alckitsampleuri://`. Anilist is supposed to call this custom protocol directly with the authorization code. But it looks like Anilist thinks I want to redirect somewhere `alckitsampleuri://` within Anilist itself and so it 404's and sends me to the senpai page. The generated URL (that works fine) looks like this.

`https://anilist.co/api/auth/authorize?grant_type=authorization_code&response_type=code&redirect_uri=alckitsampleuri://&client_id=CLIENT_ID`

but approving the client will make it 404.

**Comment by joshstar** [comment]
Yeah you mentioned it on the forums, its something I did fix but then had to revert while I was fixing a bunch of the other oAuth issues. I'll looking to add this back again.

**Comment by AndyIbanez** [comment]
Hm, it looks like it's more broken than before.

Before I could add a generic .com website and it would try to redirect to that one. Now everything seems to 404 there.

**Comment by AndyIbanez** [comment]
I got a workaround, which is to specify any random hostname after the custom scheme.

`alckitsampleuri://arandomhost.com`

This should ideally get fixed, but I will use this is a workaround for now (I had a more complicated workaround before which involved my own server, so I'm glad I got this to work this way).
